addons/auth_oauth/data/auth_oauth_data.xml

```diff
@@ -7,7 +7,7 @@
             <field name="scope">userinfo</field>
             <field name="validation_endpoint">https://accounts.odoo.com/oauth2/tokeninfo</field>
             <field name="data_endpoint"></field>
-            <field name="css_class">zocial openerp</field>
```

Comment:
Wow. Could you kill that one too: https://github.com/odoo/odoo/blob/saas-11.5/addons/auth_oauth/models/auth_oauth.py#L21 ? :)

Reply:
@qsm-odoo mmm ..  not sure about what that python class does ... 🤷‍♂️
should I remove https://github.com/odoo/odoo/blob/saas-11.5/addons/auth_oauth/models/auth_oauth.py#L21 or replace it with something like
```javascript
css_class = fields.Char(string='CSS class', default='fa fa-sign-in')
```

 ?

Reply:
You can replace the default as you suggested it is even better. I checked with JKE and zocial is an old library (which is not there anymore) which automatically created an image of the social network on page loading (so an element with "zocial google" was automatically converted to an image with the official google logo). That was before we used icons in Odoo :')

Reply:
Wau! Now I feel guilty to remove this little guy... I thought it was a typo, but it probably the last testimony of a 'fa' less era. Rest in peace little zocial guy 😥